Sources/OpenAPIRuntime/Interface/ClientTransport.swift

```diff
@@ -207,7 +207,7 @@ public protocol ClientTransport: Sendable {
 ///         ) async throws -> Response {
 ///             var request = request
 ///             request.headerFields.append(.init(
-///                 name: "Authorization", value: "Bearer \(token)"
+///                 name: "Authorization", value: "Bearer \(self.bearerToken)"
```

Comment:
```suggestion
///                 name: "Authorization", value: "Bearer \(bearerToken)"
```

We generally omit the `self.` unless required by the compiler.

Reply:
Personally I think we _should_ use explicit `self.` because it helps with local reasoning and avoids unintended shadowing.

However, @czechboy0 is right that it's not currently a convention in this repo so, if we do move to it, we'll do so holistically, across the whole codebase, with the aid of `swift-format`.

So, with that in mind, I think we should take this suggestion.